lib/content-services/src/lib/pipes/is-included.pipe.ts

```diff
@@ -21,7 +21,11 @@ import { Pipe, PipeTransform } from '@angular/core';
     name: 'adfIsIncluded'
 })
 export class IsIncludedPipe<T> implements PipeTransform {
-    transform(value: T, array: T[]): boolean {
-        return array.includes(value);
+    transform(value: T, array: T[], property?: string): boolean {
```

Comment:
can we pass function instead of property? It will give us more flexibility for more advanced comparators like comparator case insensitive or sensitive etc. We can then implement function and pass it to that pipe as 'comparator' function